app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java

```diff
@@ -468,11 +478,12 @@ private void setDefaultKeyboard() {
             supportedLocales[i] = mKeyboards.get(i).getLocale().toLanguageTag();
         }
         Locale bestMatch = localeList.getFirstMatch(supportedLocales);
-        mCurrentKeyboard = getKeyboardForLocale(bestMatch);
-        if (mCurrentKeyboard == null) {
+        keyboard = getKeyboardForLocale(bestMatch);
+        if (keyboard == null) {
             // Fall back to english.
-            mCurrentKeyboard = getKeyboardForLocale(Locale.ENGLISH);
+            keyboard = getKeyboardForLocale(Locale.GERMAN);
```

Comment:
Is this a typo? It looks like it should still be getKeyboardForLocale(Locale.ENGLISH);

Reply:
good catch, yes it was a typo (changed it for a test)